lib/metadataTypes/definitions/Automation.definition.js

```diff
@@ -46,7 +46,7 @@ module.exports = {
     createdNameField: 'createdByName',
     lastmodDateField: 'lastSavedDate',
     lastmodNameField: 'lastSavedByName',
-    restPagination: false,
+    restPagination: true,
```

Comment:
nice it does this but does it work?

Reply:
we use `super.retrieveREST` in `#getAutomationNotificationsREST()` and on a smaller scale in `retrieveForCache()`, to get the legacy ID.

- https://github.com/Accenture/sfmc-devtools/blob/e263af64d2d4495cd4c6a0545756e585689d8cf0/lib/metadataTypes/Automation.js#L115
- https://github.com/Accenture/sfmc-devtools/blob/e263af64d2d4495cd4c6a0545756e585689d8cf0/lib/metadataTypes/Automation.js#L233

those requests need pagination because they use the legacy API which only allows 50 records per page to begin with.

The main requests for Automations actually use SOAP.retrieveBulk plus a promise.all around single rest.get requests with object id in the path...